.github/workflows/statistics-daily.yml

```diff
@@ -61,16 +63,30 @@ jobs:
             echo ""
             echo "$TOP5_FLAKYTESTS" | jq -c '.[]' | while read -r top_flakytest; do
               NUMBER_OF_FLAKES=$(echo "$top_flakytest" | jq -r '.number_of_flakes')
-              TEST_CLASS_NAME=$(echo "$top_flakytest" | jq -r '.test_class_name')
+              FULLY_QUALIFIED_TEST_CLASS_NAME=$(echo "$top_flakytest" | jq -r '.test_class_name')
+              TEST_CLASS_NAME="${FULLY_QUALIFIED_TEST_CLASS_NAME##*.}"
               TEST_NAME=$(echo "$top_flakytest" | jq -r '.test_name')
-              echo "• ${NUMBER_OF_FLAKES}x \`${TEST_CLASS_NAME##*.}.$TEST_NAME\`\n"
+              SUFFIX=""
+
+              # Try to find GH issues about related flaky tests:
+              # First, search specifically by test class name + test name + "flaky"
+              GH_ISSUE_ID=$(gh issue list -R camunda/camunda --search "in:title $TEST_CLASS_NAME AND in:title $TEST_NAME AND in:title flaky" --json number --jq '.[].number' | head -n 1)
+              if [ -z "$GH_ISSUE_ID" ]; then
+                # If nothing found, search by test class name + "flaky"
+                GH_ISSUE_ID=$(gh issue list -R camunda/camunda --search "in:title $TEST_CLASS_NAME AND in:title flaky" --json number --jq '.[].number' | head -n 1)
```

Comment:
🙃 I guess could be a false positive, as this would also satisfied by class name with different test name, but I guess it is fine.

Reply:
Yea it could be. But when discovered, it can be solved by creating a ticket that's name contains both class and test name which will then take precendence :)